datafusion/functions-array/src/udf.rs

```diff
@@ -166,146 +166,6 @@ impl ScalarUDFImpl for StringToArray {
     }
 }
 
-make_udf_function!(
```

Comment:
is this just an accidental left over copy?

Reply:
Yes, there might be some other left things